### DISCUSSION TITLE
How should I mark two linters conflicting and choose one of them under the project in structure?

### DISCUSSION THREAD

**Discussion by bigshans** [w0rp/ale#4794] [Q&A - Ask for help with problems]
I'm using tsserver primarily, but I also write some deno projects sometimes. Currently, it seems that I can only completely disable one of the liners, but I want to use only Tsserver in some projects and only Deno in others. Is there any general configuration that can solve this problem?

**Comment by hsanson** [comment]
The best way to do this is to have project specific configurations. Some discussion on how to achieve this below:

- https://www.reddit.com/r/neovim/comments/tcusd9/best_plugin_for_per_project_config_changes_and/
- https://www.reddit.com/r/vim/comments/ir66yv/patterns_for_setting_perproject_filetype_settings/
- https://medium.com/@dnrvs/per-project-settings-in-nvim-fc8c8877d970
- https://andrew.stwrt.ca/posts/project-specific-vimrc/

**Comment by bigshans** [comment]
This is a way to configure it but finally, I fork this repository and hard-coded the logic. It was solved but not perfect.
